package.json

```diff
@@ -220,6 +220,7 @@
     "@types/sockjs-client": "^1.1.1",
     "@types/uuid": "8.3.0",
     "@welldone-software/why-did-you-render": "4.0.6",
+    "history": "4.10.1",
```

Comment:
added this because I got errors that createMemoryHistory was undefined in tests, but this did not fix it